fabfile/adage_server.py

```diff
@@ -210,6 +210,7 @@ def init_instance():
 def build_interface():
     """ have grunt perform a deployment build for us """
     with cd(CONFIG['interface_dir']), \
+            shell_env(PHANTOMJS_BIN='/usr/local/bin/phantomjs'), \
```

Comment:
- [ ] test AWS deployment with the local install of `phantoms` under `node_modules` noted above